pkg/syncer/metadatasyncer.go

```diff
@@ -91,16 +92,15 @@ var (
 	nodeMgr node.Manager
 	// isPodVMOnStretchSupervisorFSSEnabled is true when PodVMOnStretchedSupervisor FSS is enabled.
 	isPodVMOnStretchSupervisorFSSEnabled bool
-
-	// For core API resource ResourceAPIGroup will be set to "". PVC is part of the core API resources
-	// and ResourceAPIgroupPVC is set to "".
-	ResourceAPIgroupPVC = ""
 )
 
 const (
 	ResourceKindPVC           = "PersistentVolumeClaim"
 	QuotaExtensionServiceName = "volume.cns.vsphere.vmware.com"
 	scParamStoragePolicyID    = "storagePolicyID"
+	// ResourceAPIgroupPVC is the API group PVCs belong to. PVC is part of the core API resources
+	// which are denoted as "".
+	ResourceAPIgroupPVC = ""
```

Comment:
why ResourceAPIgroupPVC is empty string?

Reply:
The APIGroup for PVC is empty string